### PR TITLE
Unified Hardsuits, Helmet is automatically attached to hardsuits

### PR DESCRIPTION
**Pull request by Couls**
**What does this PR do:**
Hardsuit helmets are now automatically attached when the item is created, removing the need to attach it.

**Changelog:**
:cl:
add: Hardsuits now come with their helmets
/:cl:

### CODE REVIEW

**Comment by AffectedArc07** [comment]
Travis failure:
image
Failure reason: Unable to download BYOND
Possible causes: Skiddies DDOSing BYOND again

**Comment by FreeStylaLT** [comment]
The technology.. It's finally there..

**Comment by FlattestGuitar** [comment]
This but unironically.

File: paradise.dme [hunk 1]
```diff
@@ -2447,4 +2447,4 @@
 #include "goon\code\datums\browserOutput.dm"
 #include "interface\interface.dm"
 #include "interface\skin.dmf"
-// END_INCLUDE
\ No newline at end of file
+// END_INCLUDE
```
Comment (AffectedArc07) [inline paradise.dme]: wow cant believe you changed the dme /s

**Comment by Couls** [comment]
Fox has offered to rework helmets instead

**Pull request closed without merging**